
#Parameters for the espeleo_planning package

# Curve number
N_curve: 0
# 0 -> Generic parametric equation
# 1 -> Ellipse
# 2 -> Eigth like curve
# 3 -> Square like curve
# 4 -> Sine curve
# 5 -> Smooth eigth like curve
# 6 -> Amoeba like

# Number of points to be sampled
N_points: 200

# Stretch factor of the curve in the x direction
a: 1.5
# Stretch factor of the curve in the y direction
b: 1.0
# Rotation of the curve around the z axis (in dregrees)
phi: 0.0
# Displacement in the x direction (meters)
cx: 0.0
# Displacement in the y direction (meters)
cy: 0.0



#These string parameters respect the python syntax - only used when N_curve = 0
# u_i: 0
# u_f: 1
# equation: "[2*cos(2*pi*u), 1.5*sin(2*pi*u), 0.0]"
# equation: "[2*cos(2*pi*u)*sin(2*pi*u), 1.5*sin(2*pi*u), 0.0]"

u_i: 0
u_f: 6.283185307179586
# equation: "[2*(u-pi), 0, 0.0]"
# quation: "[2*cos(u), 2.0*sin(u), 0.2*sin(u)]"
equation: "[2*cos(u), 2.0*sin(u), 0.2*cos(6*u)]"
# equation: "[2*cos(u)*sin(u), 1.5*sin(u), cos(1*u)/5.0]"
# equation: "[2*cos(u_trick[0]), 2.0*sin(u_trick[0]), 0.0]"


#Flag to indicate if the path is closed or not
closed_path_flag: True
#Number of points to be inserted in between each pais of points of the received path
insert_n_points: 1
#Number of points to use in the average filter (it is forced to be an odd number) - if 0 the path is not filtered
filter_path_n_average: 1

